examples/calculator.js

```diff
@@ -16,7 +16,31 @@
 
 const assert = require('assert');
 const bacardi = require('bindings')('bacardi.node');
-const calculator = new bacardi.Calculator();
+const chai = require('chai');
```

Comment:
We already use Mocha as testing framework. So, you can just use the APIs.
Do you have any reason that we have to add this new testing framework?

Reply:
but mocha seems to do not support expect() function.

So I think we cannot handling expect-exception logic test like chai.